### DISCUSSION TITLE
centering of stimuli and text

### DISCUSSION THREAD

**Discussion by SSchwoebel** [jspsych/jsPsych#266] [Q&A]
Hello,

I am using jsPsych for an experiment, but I don't seem to be able to get my stimuli and text centered on my screen. I looked into the .css file and modified it, but it looks like it already contains centering, but it doesn't seem to work. Where can I configure that, and how?

**Comment by jodeleeuw** [comment]
The CSS is a mess right now. Part of the goal for the next release is to really clean it up. In the latest version, there is an option to center everything by default.

For now the best option is to modify the CSS as needed to fit your experiment. You might be able to use the [.center-content and .block-center](https://github.com/jodeleeuw/jsPsych/blob/v5.0.3/css/jspsych.css#L45-L52) styles as needed.

**Comment by jodeleeuw** [comment]
I'm closing this, since the CSS revisions are covered by #163 and #244
